packages/app/src/cli/models/extensions/specifications/marketing_activity_extension.ts

```diff
@@ -0,0 +1,66 @@
+import {createExtensionSpecification} from '../specification.js'
+import {BaseSchema} from '../schemas.js'
+import {zod} from '@shopify/cli-kit/node/schema'
+
+const MarketingActivityExtensionSchema = BaseSchema.extend({
+  title: zod.string().min(1),
```

Comment:
You mentioned in this [doc](https://docs.google.com/document/d/1KwE1Q7AzPULXAucOrCd84h5P8ISY1Mc2WT9woW3gKDw/edit) that this validation will move to Core. Is this still needed?

Reply:
looks like it's still being built so validation in cli is good for now: https://github.com/Shopify/ce-customer-behaviour/issues/5232#issuecomment-2161137710